br/pkg/storage/s3.go

```diff
@@ -70,6 +70,9 @@ var permissionCheckFn = map[Permission]func(*s3.S3, *backuppb.S3) error{
 	GetObject:     getObject,
 }
 
+// WriteBufferSize is the size of the buffer used for writing. (64K may be a better choice)
+var WriteBufferSize = 5 * 1024 * 1024
```

Comment:
Can we merge WriteBufferSize and hardcodedS3ChunkSize ?

Reply:
There are not same.